dbt-cta/quickbooks/models/0_ctes/bills_Line_ItemBasedExpenseLineDetail_ab2.sql

```diff
@@ -0,0 +1,21 @@
+{{ config(
+    cluster_by = "_airbyte_emitted_at",
+    partition_by = {"field": "_airbyte_emitted_at", "data_type": "timestamp", "granularity": "day"},
+    unique_key = "_airbyte_ab_id"
+) }}
+-- SQL model to cast each column to its adequate SQL type converted from the JSON schema type
+-- depends_on: {{ ref('bills_Line_ItemBasedExpenseLineDetail_ab1') }}
+select
+    _airbyte_Line_hashid,
+    cast(UnitPrice as {{ dbt_utils.type_float() }}) as UnitPrice,
```

Comment:
This field was failing with the autogenerated airbyte dbt. It was originally set as an int. It was trying to cast a number like `30.5` and throwing an error. It definitely seems like it is always a currency. I changed every instance of `UnitPrice` to a float as it seems like it's a currency for all of them. We were just getting one error because the other instances weren't populated in our pulled data.

Reply:
This sounds like the correct approach to me! Gotta love how Airbyte's default dbt is like at least 99% fine (well except for the nested fields like Jesus mentioned lol) except for when it's not 🫠